### ISSUE TITLE
$RequestData renders incorrectly

### ISSUE THREAD

**Issue by dhjgodin** [frhagn/Typewriter#198]
I have the controller method 

```
[HttpDelete]
[Route("list/{fooId}/{barId}")]
public async Task DeleteFooBar(int fooId, Guid BarId)
```

My typewriter code is
`return this.http.$HttpMethod(`$Url`, $RequestData)`

It now renders

`return this.http.delete(`list/${fooId}/${encodeURIComponent(barId)}`, barId)`
instead of 
`return this.http.delete(`list/${fooId}/${encodeURIComponent(barId)}`, null)`

And no longer compiles.

**Comment by frhagn** [comment]
Hi @dhjgodin, sorry about that.. 
I've published a new version (1.12.2) that should resolve this issue, could you please confirm? Thanks

**Comment by dhjgodin** [comment]
Yep that's working now. Cheers for the quick turn around.
